roles/lokole/README.rst

```diff
@@ -14,9 +14,56 @@ The Lokole is a simple email client that offers functionality like:
 Using It
 --------
 
-Lokole should be accessible at http://box/lokole
+The Lokole web app can be accessed at http://box/lokole.
+
+Administration
+--------------
+
+Every Lokole installation has an admin account named ``admin``. By default, the password is ``changeme1``.
```

Comment:
@holta: Done in https://github.com/ascoderu/iiab/commit/96b8dfa4d6fadd6fb81a6fe32de728916aae3e07.

@aidan-fitz: I don't have the IIAB environment set up so could you please verify the commit so that we can submit a pull request? Thanks!

Reply:
@c-w: Code-reviewing it now. Looks good to me!

Reply:
Submitted https://github.com/iiab/iiab/pull/1317.